winml/test/common/googleTestMacros.h

```diff
@@ -84,26 +85,22 @@
 #define WINML_EXPECT_HRESULT_FAILED(hresult_expression) EXPECT_HRESULT_FAILED(hresult_expression)
 #define WINML_EXPECT_THROW_SPECIFIC(statement, exception, condition) EXPECT_THROW_SPECIFIC(statement, exception, condition)
 
+
+#define RUNTIME_PARAMETER_EXISTS(param)                          \
+  auto no_gpu_tests = RuntimeParameters::Parameters.find(param); \
+  no_gpu_tests != RuntimeParameters::Parameters.end() && no_gpu_tests->second != "0"
```

Comment:
Can be a function instead of a macro

Reply:
Wasn't the whole point of having these macros so that they could conditionally compiled for either google test or taef test based on which repo we build in?

Reply:
Updated to use functions instead of macros for test filtering

Reply:
The other ones had to return or skip from the test function, so the return statement had to be in the function and had to be implemented as a macro